src/scenes/FunderReport/MetricBox.tsx

```diff
@@ -78,7 +78,20 @@ const MetricBox = ({ metric, index, year, quarter, lastIndex }: MetricBoxProps)
           <Typography fontWeight={600} fontSize='16px' marginTop={1}>
             {strings.PROGRESS_NOTES}
           </Typography>
-          <Typography fontWeight={400}>{metric.progressNotes}</Typography>
+          <Textfield
+            display={true}
```

Comment:
```suggestion
            display
```

`={true}` can be removed for the sake of consistency since it isn't being included for `preserveNewLines`